ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Overview/ClustersTable.tsx

```diff
@@ -1,48 +1,147 @@
 import React from 'react';
-import { Bullseye, Spinner } from '@patternfly/react-core';
+import { pluralize } from '@patternfly/react-core';
+import { Table, Thead, Tr, Th, Td, Tbody } from '@patternfly/react-table';
+import { Link } from 'react-router-dom';
+import { gql, useQuery } from '@apollo/client';
 
-import TableErrorComponent from 'Components/PatternFly/TableErrorComponent';
 import useURLPagination from 'hooks/useURLPagination';
+import { ClusterType } from 'types/cluster.proto';
+import {
+    TbodyLoading,
+    TbodyError,
+    TbodyEmpty,
+    TbodyFilteredEmpty,
+} from 'Components/TableStateTemplates';
+import { getTableUIState } from 'utils/getTableUIState';
+import { ensureExhaustive } from 'utils/type.utils';
 
+import { DynamicColumnIcon } from '../../components/DynamicIcon';
+import { getPlatformEntityPagePath, getRegexScopedQueryString } from '../../utils/searchUtils';
 import { QuerySearchFilter } from '../../types';
 
+function displayClusterType(type: ClusterType): string {
+    switch (type) {
+        case 'GENERIC_CLUSTER':
+            return 'Generic';
+        case 'KUBERNETES_CLUSTER':
+            return 'Kubernetes';
+        case 'OPENSHIFT_CLUSTER':
```

Comment:
From what I understand, this case is essentially deprecated but we'll handle it here anyway.

Reply:
OpenShift 3.11 will never die.

https://xkcd.com/1782/